src/lib/Tab/LocationView/RelationsTab.php

```diff
@@ -146,7 +146,7 @@ public function getTemplateParameters(array $contextParameters = []): array
 
         $viewParameters['relations'] = $relations;
 
-        if (true === $this->permissionResolver->hasAccess('content', 'reverserelatedlist')) {
+        if ($this->permissionResolver->hasAccess('content', 'reverserelatedlist')) {
```

Comment:
If there were role assignment level limitations excluding this right for the given context, it would return true-ish value even if a user had no actual permission. Unlikely to happen, since the entire content would not be accessible, but still, the correct check is:
```suggestion
        if ($this->permissionResolver->canUser('content', 'reverserelatedlist', $content)) {
```
cc @mikadamczyk